css/my_website.css

```diff
@@ -0,0 +1,36 @@
+body {
+    margin: 0px;
+    background-color: rgb(231, 231, 231);
+    color: rgb(65, 65, 65);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    line-height: normal;
+}
+
+header {
+    background-color: rgb(28, 30, 155);
+    color: #ffffff;
+    height: 75px;
+    padding: 0em;
+    width: 100%;
+    overflow: hidden;
+}
+
+p {
+    color: black;
+    padding-top: .5em;
+    padding: 1em;
+    margin-left: 1em;
+    margin-right: 1em;
+    background-color: white;
+}
+
+header h1 {
+    margin: 5px;
+    padding: 10px;
+    padding-left: .5em;
+}
+
+img {
+    padding: 1em;
+}
```

Comment:
This is looking good. You might want to organize these a bit though. Perhaps start body, p, imp, header, header h1? It seems odd to have p between header and header h1.

Reply:
You're right I will get right on that!